docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -0,0 +1,27 @@
+[id='Security-Strategy-Keycloak-{chapter}']
+= Security Strategy for {WFM-RC-NameShort} Keycloak
+
+== Session Management
+Keycloak uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
+Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
+session options.
+One of the key functions of Keycloak is you can administer sessions on the
+link:https://keycloak.gitbooks.io/documentation/server_admin/topics/sessions/administering.html[Keycloak server]
+
+== Access Control
+Keycloak has role based access control. This can be configured on the Keycloak server see
+link:https://keycloak.gitbooks.io/documentation/server_admin/topics/roles.html[roles administration] for more information.
+
+== Password Storage
+With the default configuration of Keycloak passwords are stored on the Keycloak server. Access to user passwords is restricted.
```

Comment:
Passwords hashes (never actual passwords) - We can find some proper sentence from keycloak docs to avoid such language mistakes that can have huge legal implications.

Reply:
Password Policy documentation covers this . Will reword.